### PR TITLE
the Hello Microverse output

### PR DESCRIPTION
**Pull request by Said-laasri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
```
Comment (youmari) [inline README.md:28]: - [ ] Also please try to add how can other use your project in their machines like how to install and how to run the projects locally and try to remove the empty headers from know like `Run tests`

**Review by youmari — COMMENTED** [review]
Hi @Said-laasri  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Nice implementing the requirements
✔️ Github flow 
## Required Changes ♻️

- [x] please next time don't merge your Pull Request until it gets approved that will be better
- [ ] Please try to add a descriptive short summary telling in more detail what you've done in the Pull Request
- [ ] I encourage you to follow the professional rules to keep your PR pro by committing a descriptive commits messages next time 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Said-laasri** [comment]
hi
sorry for the rush to merge my bad.

i will change everything you asked to improve. then submit again .

thanks

File: README.md (hunk 1 above)
Comment (Said-laasri) [reply README.md:28]: so for how to . do i say for example like pull the code and use your browser to open the hrtml file to get the result??!

thanks alot again

File: README.md [hunk 2]
```diff
@@ -1,2 +1,68 @@
...
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
[the remark this answers is not in the document]
Comment (Said-laasri) [reply README.md:48]: yes i miss this one bad my fault  .
thanks for review.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,68 @@
-# Hello-world
-Hello Microverse project
+
+# Project Name
+Hello Microverse
+> Description the project.
```
[the remark this answers is not in the document]
Comment (youmari) [reply README.md:4]: @Said-laasri ghir ktab chnu project kydit bhal daba `this is a hello microverse project` where I using standard linters and how to open a professional Pull request

File: README.md (hunk 1 above)
Comment (Said-laasri) [reply README.md:28]: @youmari

**Comment by Said-laasri** [comment]
@youmari  please review my readme.md file 
few changes were added please check it .thanks

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (danmainah) [reply README.md:48]: - [ ] Kindly add your information in your readme;
- GitHub: [@githubhandle](https://github.com/githubhandle)
- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)

Currently, the template above is displayed please replace the name and link with your own information.

**Review by danmainah — COMMENTED** [review]
Hi @Said-laasri, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ Github flow was correctly used
✔️  Good design

 # Required Changes ♻️

- [ ] Kindly add a pull request description as requested,the image below show the section to add the description 
image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Said-laasri) [reply README.md:28]: https://github.com/Said-laasri/Hello-world/blob/main/README.md
hi I did change the readme file please check again ???

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Said-laasri) [reply README.md:48]: https://github.com/Said-laasri/Hello-world/blob/main/README.md
please check what is the problem ???
@danmainah

File: README.md (hunk 1 above)
Comment (Said-laasri) [reply README.md:28]: @danmainah i changed the file before

**Pull request merged**